### PR TITLE
Added doc for v1.1

### PR DESCRIPTION
**Pull request by anikesh-butler**
```javascript
  butlerBlocks.createDocLabeler('ButlerDocumentLabeler', data, {
    onSaveCallback,
    // saveActionButtonText: 'Confirm',
    // fieldDisplayNameFormatter: (fieldName) => fieldName.toUpperCase(),
    // displayOnly: true,
    // hideSaveButton: true,
    // onLabelUpdate: (docInfo: DocumentLabelerOutputDataDto) => {
    //  console.log('Label changed', docInfo)
    // }
  });
```

### New config object API

| Param          | Type     | Required | Default | Description                                                                               |
|----------------|----------|----------|---------|----------------------------------------------------------------------------------|
| onSaveCallback | Function | <li> [x] </li>     | None |Method called after clicking the save button with latest label changes |
| onLabelUpdate  | Function | <li> [ ]  </li>    | undefined | Callback method used to listen for label changes. can be used to implement custom save UI |
| hideSaveButton | Boolean  | <li> [ ] </li>     | false | Removes the save button from the UI |
| displayOnly    | Boolean  | <li> [ ] </li>     | false | Toggle display only model which removes all the edit and save UI elements |
| fieldDisplayNameFormatter | Function  | <li> [ ] </li>  | undefined    | Used to format or change the field name display |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -111,9 +115,30 @@ const initializeDocLabeler = async ({ modelId, documentId }) => {
...
+    // }
+  });
 };
 
 // Call this function when you want to display the labeler!
 initializeDocLabeler(myDocument);
 ```
+
+
+### New config object api
```
Comment (butler-matt) [inline README.md:135]: Should we name this section something like:
"Configuration Option Details"?

File: docs/version_1_0.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
+# Butler Blocks v1.0
+
+## Install Butler Blocks
+
+```cli
+npm install butler-blocks@1.0.6 --save
```
Comment (butler-matt) [inline docs/version_1_0.md:6]: Should we also show an example using yarn as well?

**Review by butler-matt — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (anikesh-butler) [reply README.md:135]: Changing to `Configuration Option Api` as it's technically an API of a specific object.

File: docs/version_1_0.md (hunk 2 above)
Comment (anikesh-butler) [reply docs/version_1_0.md]: Done

**Pull request merged**